resources/qml/PrintSetupSelector/PrintSetupSelectorHeader.qml

```diff
@@ -15,6 +15,7 @@ RowLayout
 
     Cura.IconWithText
     {
+        id: printSetupSelectorProfileText
```

Comment:
It doesn't indicate the profile but the layer height here. I suggest changing it to `printSetupSelectorLayerHeightText`, but you'd have to change that in your test as well.